.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,55 @@
+name: "Step 2: Publish Release"
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The target branch"
+        required: false
+      release_url:
+        description: "The URL of the draft GitHub release"
+        required: false
+      steps_to_skip:
+        description: "Comma separated list of steps to skip"
+        required: false
+        default: "ensure-sha"
```

Comment:
We can remove this default. It's only needed in JupyterLab for now: https://github.com/jupyterlab/jupyterlab/issues/13442

```suggestion
```